Cargo.toml

```diff
@@ -116,3 +116,4 @@ vergen = "7"
 [profile.release]
 codegen-units = 1
 lto = true
+strip = true
```

Comment:
Oh, nice, looks like the `true` value [strips all symbols](https://doc.rust-lang.org/cargo/reference/profiles.html#strip) as well.